pkg/metrics/metrics.go

```diff
@@ -53,6 +54,15 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 		[]string{"driver_name", "method_name", "grpc_status_code", "disk_type", "enable_confidential_storage", "enable_storage_pools"})
+
+	mountErrorMetric = metrics.NewCounterVec(&metrics.CounterOpts{
+		Subsystem:      "node",
+		Name:           "mount_errors",
+		Help:           "Node server file system mounting errors",
+		StabilityLevel: metrics.ALPHA,
+	},
+		[]string{"error_type"},
```

Comment:
Do we have a SLO design doc for this metrics? We probably also want info of "driver_name", "grpc_status_code", "disk_type" etc. extra information?

Reply:
The thing is we need success calls and error calls to calculate the right SLO, so we probably need to emit both.

Reply:
Check out existing csi/k8s volume metrics to follow similar patterns.